### PR TITLE
[GUI Editor] Tweaks and Fixes for Demo

### PR DESCRIPTION
**Pull request by msDestiny14**
-Added more controls. 
-Fixed various bugs.
-Removed TODO statements. All issues can now be tracked here: https://github.com/BabylonJS/Babylon.js/issues/9811

### CODE REVIEW

File: guiEditor/src/components/propertyTab/propertyTabComponent.tsx [hunk 1]
```diff
@@ -241,12 +241,20 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
     render() {
         if (this.state.currentNode) {
             return (
-                <div id="propertyTab">
-                    <div id="header">
+                <div id="propertyTab"                                 onFocus={() => {(this.props.globalState.blockKeyboardEvents = true); console.log("blocked");}}
```
Comment (Popov72) [inline guiEditor/src/components/propertyTab/propertyTabComponent.tsx:244]: I think you should add "Remove console.log(...)" in https://github.com/BabylonJS/Babylon.js/issues/9811, else you may forget that one ;)

File: guiEditor/src/guiNodeTools.ts [hunk 2]
```diff
@@ -40,11 +46,42 @@ export class GUINodeTools {
...
                 return element;
+            case "ImageButton":
+                element = Button.CreateImageButton("Button", "Click Me", "textures/grass.png");
+                break;
+            case "VirtualKeyboard":
+                element = new VirtualKeyboard();
+                element.addKeysRow(["1","2", "3","\u2190"]);
+                break;
+            case "Image": 
+                element = new Image("Image", "https://i.imgur.com/ntIgFT6.jpg");
```
Comment (Popov72) [inline guiEditor/src/guiNodeTools.ts:57]: Are the pictures on imgur available forever? I guess you should instead point to a local Babylon texture, as you did for the image button.

File: guiEditor/src/components/guiList/guiListComponent.tsx [hunk 3]
```diff
@@ -40,10 +40,10 @@ export class GuiListComponent extends React.Component<IGuiListComponentProps, {
         // Block types used to create the menu from
         const allBlocks: any = {
             Buttons: ["TextButton", "ImageButton"],
-            Controls: ["Slider", "Checkbox", "ColorPicker", "VisualKeyboard"],
-            Containers: ["DisplayGrid", "Grid", "StackPanel"],
+            Controls: ["Slider", "Checkbox", "ColorPicker", "VirtualKeyboard"],
+            Containers: ["DisplayGrid", "Grid"],
```
Comment (Popov72) [inline guiEditor/src/components/guiList/guiListComponent.tsx:44]: You removed `StackPanel`: we won't be able to add stack panels in a GUI?

File: guiEditor/src/workbenchEditor.tsx [hunk 4]
```diff
@@ -84,14 +64,13 @@ export class WorkbenchEditor extends React.Component<IGraphEditorProps, IGraphEd
             (evt) => {
                 if ((evt.keyCode === 46 || evt.keyCode === 8) && !this.props.globalState.blockKeyboardEvents) {
                     // Delete
-                    let selectedItems = this._workbenchCanvas.selectedGuiNodes;
+                    /*let selectedItems = this._workbenchCanvas.selectedGuiNodes;
```
Comment (RaananW) [inline guiEditor/src/workbenchEditor.tsx:67]: Why not remove it completely?

**Review by RaananW — COMMENTED** [review]
I see a lot of changes from `graph` to `workbench`, but i can't find the change in the HTML that corresponds to it. Was it already commited?

File: guiEditor/src/components/guiList/guiListComponent.tsx (hunk 3 above)
Comment (msDestiny14) [reply guiEditor/src/components/guiList/guiListComponent.tsx:44]: Not for todays demo sadly. It's not implemented properly yet.

**Comment by msDestiny14** [comment]
@RaananW yes these were bugs that I discovered not matching the HTML.

**Review by RaananW — APPROVED** [review]

**Pull request merged**